.changes/v3.11.0/1140-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Fix organization retrieval in `vcd_resource_list` when users fill the "parent" field instead of "org" [GH-1140]
```

Comment:
Changelog is missing a note about fixing retrieval of the "org" field from the provider block.

Reply:
BTW, "parent" and "org" should be quoted in code-quotes :)

Reply:
fixed